### ISSUE TITLE
Early Game Progression for BDB 1.7

### ISSUE THREAD

**Issue by CobaltWolf** [CobaltWolf/Bluedog-Design-Bureau#588]
General issue for tracking discussion of early game progression in the early game revamp update. I want to know if people have tech tree placement concerns, if there are parts people want, etc.

**Comment by CobaltWolf** [comment]
Start this off - I'd like BDB to be balanced around reduced science gains. This is good for the whole tech tree - slowing things down, giving you a chance to do crewed orbital stuff in parallel with interplanetary flybys, for example. 

But, specifically, I think this will help a lot with the early tree progression. If we basically can assume people are using reduced science gains, then we can assume that, in order to progress, they might actually HAVE to do stuff like Thor-Able or Juno II munar flybys, etc. Basically make the user stretch out their early rocket capabilities to their max in order to get to more "operational" LVs.

**Comment by IronCretin** [comment]
Are you balancing around 2.5x/JNSQ scale?

**Comment by CobaltWolf** [comment]
Yes.

**Comment by KerbalKore** [comment]
Balancing with KCT in mind might help with the slowing things down part.

**Comment by zorg2044** [comment]
Anyone have any thoughts on Castor I & II positioning? 

Right now Castor 1 unlocks in general rocketry and castor 2 subtype in advanced rocketry

**Comment by CobaltWolf** [comment]
you could almost move them each back a node honestly. Just need to make sure the radial decoupler is also available. maybe leave that in engineering101?

**Comment by Pappystein** [comment]
So Maybe I am being obtuse here (and apologies if I am) but HOW are you slowing down Science Gain?   

I am all for stretching the Tree out / somehow altering it to improve game play.   But Short of a DIFFERENT tree or multiple Mods to alter how the game grants things like Science and Funds....   I don't see how this can happen.  

As I see it three problems occur:

1. Stock Contract System would need overhaul/direct replacement.. One that eliminates many of the "Flying" Science missions as they require Aircraft not part of the Mod...  Slowed down science means they are wasting slots for missions since Aircraft are not a thing at the start of the game.
2. All Science Experiments would need to be NERFED or otherwise replaced... This includes 3rd Party... I have ideas on this subject.
3. Co-operation with other mod creators would be needed

A Simple way to SIMULATE this with the dev version is DMagic's Contract Reward Modifier.   I would suggest something like an Increase in Fund-age reward by 150% and a reduction of Science reward by 50% (so Funds are doubled and Science is halved.)   You need Funds to fly the extra rockets you will be fireing just to gain more science....

**Comment by DeltaDizzy** [comment]
> So Maybe I am being obtuse here (and apologies if I am) but HOW are you slowing down Science Gain?

You can lower science rewards in the savegame difficulty options.
